pkg/kubectl/cmd/util/helpers.go

```diff
@@ -46,6 +46,7 @@ import (
 
 const (
 	ApplyAnnotationsFlag = "save-config"
+	AddOnLabel           = "kubernetes.io/cluster-service"
```

Comment:
in my opinion, kubectl should be checking a generic annotation that a resource is write-protected. If we use that to protect cluster services, that is fine, but I prefer the annotation and kubectl flag be generic. Otherwise, we'll get people adding `kubernetes.io/cluster-service=true` just to get write protection behavior